### PR TITLE
Feature - small JSON serialization improvements

### PR DESCRIPTION
**Pull request by stijnmoreels**
Some JSON serialization settings that were present in the Azure Functions HTTP trigger, but not in the web API.

### CODE REVIEW

File: src/Arcus.Templates.WebApi/Startup.cs [hunk 1]
```diff
@@ -247,17 +247,21 @@ private static void RestrictToJsonContentType(MvcOptions options)
             options.OutputFormatters.RemoveType<StringOutputFormatter>();
         }
 
-        private static void AddEnumAsStringRepresentation(MvcOptions options)
+        private static void ConfigureJsonFormatters(MvcOptions options)
         {
             var onlyJsonInputFormatters = options.InputFormatters.OfType<SystemTextJsonInputFormatter>();
             foreach (SystemTextJsonInputFormatter inputFormatter in onlyJsonInputFormatters)
             {
+                inputFormatter.SerializerOptions.MaxDepth = 10;
```
Comment (tomkerkhove) [inline src/Arcus.Templates.WebApi/Startup.cs:255]: Should we do this?

Comment (stijnmoreels) [reply src/Arcus.Templates.WebApi/Startup.cs:255]: I'm more of the opinion to have this in place and that the developer change it to something higher if they need it, then to leave it open to 64 by default and maybe overload something.

Comment (tomkerkhove) [reply src/Arcus.Templates.WebApi/Startup.cs:255]: WDYT @fgheysels ?

Comment (fgheysels) [reply src/Arcus.Templates.WebApi/Startup.cs:255]: In my humble opinion, we should stick to the defaults defined in .NET.
Target audience is developers, so we shouldn't be too restrictive or 'parental' (or whatever you call it :) ).

Comment (tomkerkhove) [reply src/Arcus.Templates.WebApi/Startup.cs:255]: Frankly, I was thinking the same on this one

Comment (stijnmoreels) [reply src/Arcus.Templates.WebApi/Startup.cs:255]: Ok, then should we also remove it in the Azure Function HTTP trigger 'cause it's defined there too. I can do that in this PR too.

Comment (stijnmoreels) [reply src/Arcus.Templates.WebApi/Startup.cs]: Left the `IgnoreNullValues` still there, but removed the `MaxDepth` in both the web API as the Azure Functions HTTP trigger template. Resolving.

**Review by tomkerkhove — APPROVED** [review]

**Pull request merged**